src/main/java/de/tum/in/www1/artemis/repository/SubmissionRepository.java

```diff
@@ -26,6 +26,10 @@
     @Query("select submission from Submission submission where type(submission) in (ModelingSubmission, TextSubmission) and submission.submitted = false and not submission.participation is null")
     List<Submission> findAllUnsubmittedModelingAndTextSubmissions();
 
+    /* Get all submissions from a participation_id */
+    @Query("SELECT submission FROM  Submission submission WHERE submission.participation.id = :#{#participationId} ")
```

Comment:
This could be a standard jpa query without an explicit query:

`List<Submission> findAllByParticipationId(Long participationId);`